### PR TITLE
Cleaned up README and added startpage

### PR DESCRIPTION
**Pull request by codepo8**
This change cleans out the README of outdated and overly complicated information and adds new screencasts and screenshots instead.
There's also a new optional startpage to show when opening the tools.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -188,107 +178,33 @@ To add a new debug configuration, in your `launch.json` add a new debug config w
...
 * `sourceMaps`: By default, the extension will use sourcemaps and your original sources whenever possible. You can disable this by setting `sourceMaps` to false.
 * `pathMapping`: This property takes a mapping of URL paths to local paths, to give you more flexibility in how URLs are resolved to local files. `"webRoot": "${workspaceFolder}"` is just shorthand for a pathMapping like `{ "/": "${workspaceFolder}" }`.
-* `sourceMapPathOverrides`: A mapping of source paths from the sourcemap, to the locations of these sources on disk. See [Sourcemaps](#sourcemaps) for more information
+* `sourceMapPathOverrides`: A mapping of source paths from the sourcemap, to the locations of these sources on disk. 
 * `urlFilter`: A string that can contain wildcards that will be used for finding a browser target, for example, "localhost:*/app" will match either "http://localhost:123/app" or "http://localhost:456/app", but not "https://stackoverflow.com". This property will only be used if `url` and `file` are not specified.
 * `timeout`: The number of milliseconds that the Microsoft Edge Tools will keep trying to attach to the browser before timing out. Defaults to 10000ms.
 
-#### Sourcemaps
```
Comment (mliao95) [inline README.md:195]: Maybe move to a `SourceMaps.md` file and reference it where necessary

File: README.md [hunk 2]
```diff
@@ -3,93 +3,89 @@
...
 
-* Debug configurations for launching Microsoft Edge browser in remote-debugging mode and auto attaching the tools.
-* Side Bar view for listing all the debuggable targets, including tabs, extensions, service workers, etc.
 * Fully featured Elements and Network tool with views for HTML, CSS, accessibility and more.
 * Screen-casting feature to allow you to see your page without leaving Visual Studio Code.
-* Go directly to the line/column for source files in your workspace when clicking on a link or CSS rule inside the Elements tool.
-* Auto attach the Microsoft Edge Tools when you start debugging with the [Debugger for Microsoft Edge](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-edge) extension.
+* Debug configurations for launching Microsoft Edge browser in remote-debugging mode and auto attaching the tools.
 * Debug using a windowed or headless version of the Microsoft Edge Browser
+* Side Bar view for listing all the debuggable targets, including tabs, extensions, service workers, etc.
```
Comment (mliao95) [inline README.md:43]: As a note, service and shared workers are hidden from the target list by default and can be enabled via settings.  Up to you whether we want to publicize a disabled-by-default functionality.

**Review by mliao95 — COMMENTED** [review]
Left some comments.

File: README.md [hunk 3]
```diff
@@ -109,45 +105,39 @@ Attach the extension to your live reloading project
 * Navigate in the browser of the extension to `localhost:8080/{file name you want to open}`	
 * All changes that are saved in this folder now trigger a refresh
 
-#### Opening source files from the Elements tool
+## Changing Extension Settings
```
[the remark this answers is not in the document]
Comment (codepo8) [reply README.md:108]: Not in the current version, yet. 
image

File: README.md [hunk 4]
```diff
@@ -109,45 +105,39 @@ Attach the extension to your live reloading project
...
+You can customize the extension to your needs. You can reach the settings by clicking the gear icon of the extension listing or via the settings menu.
 
-![Microsoft Edge Tools - Links](links.png)
+![The gear icon in the extension listing](img/edge-devtools-for-vscode-settings.png)
 
-The source files for these applied styles and attached event handlers appear in the form of links to a url specified by the browser. Clicking on one will attempt to open that file inside the Visual Studio Code editor window. Correctly mapping these runtime locations to actual files on disk that are part of your current workspace, may require you to enable source maps as part of your build environment.
+### Turning on Network Inspection
 
-An example webpack configuration for sass and typescript is given below:
+You can enable the Network Pane to inspect any network request of the attached browser. To do this, change the setting and restart the extension.
```
[the remark this answers is not in the document]
Comment (codepo8) [reply README.md:116]: Fixing.

[the remark this answers is not in the document]
Comment (codepo8) [reply README.md:116]: Fixed and removed the extra screencast

File: README.md (hunk 1 above)
Comment (codepo8) [reply README.md:195]: Done.

File: README.md (hunk 2 above)
Comment (codepo8) [reply README.md:43]: Let's not for now, and maybe think about how we could make that more interesting. As we don't have the applications tab for debugging service workers it seems superfluous.

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (codepo8) [reply README.md:108]: Added the screencast and changed the text accordingly.

**Review by mliao95 — APPROVED** [review]
LGTM.  The new pages look great!  Thanks!

**Review by vidorteg — APPROVED** [review]
looks great!

**Pull request merged**